src/indexCors.ts

```diff
@@ -12,8 +12,9 @@ app.use(express.json())
 app.use(cors())
 
 app.all('*', async (req: Request, res: Response) => {
-  const { body, headers, method, path } = req
+  const { body, headers, ip, method, path } = req
   const { 'proxy-url': proxyUrl } = headers
+  const ipString = ip.includes(':') ? `[${ip}]` : ip
```

Comment:
Hmm, my only wonder is if we're accounting for existing forwarded headers here. I believe express has an array of IPs that gives the request if there exists already a series of IPs in the forwarded header. We should be appending/prepending (can't remember which) to that.

Reply:
fixed